src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -1789,8 +1848,19 @@ public override TypeKind TypeKind
                                     break;
 
                                 case SpecialType.System_ValueType:
-                                    // System.Enum is the only class that derives from ValueType
-                                    if (this.SpecialType != SpecialType.System_Enum)
+                                    if (this.SpecialType == SpecialType.System_Enum)
+                                    {
+                                        // System.Enum is the only class that derives from ValueType
```

Comment:
>// System.Enum is the only class that derives from ValueType [](http://example.com/codeflow?start=40&length=60)

This comment seems to be no longer true.

Reply:
> This comment seems to be no longer true.

Why? It is talking about classes in metadata.